book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly ensure that `Getters and Setters for @rentals is present`. See suggestion below:

```suggestion
  attr_accessor :title, :author,  :rentals
```